typeclasses/baseobject.py

```diff
@@ -0,0 +1,19 @@
+from evennia import DefaultObject
```

Comment:
These properties are technically not necessary, but they make other code prettier, so I'm fine with these properties existing. However, this should be using super so all of the below multiple inheritances are not necessary. Multiple inheritance should be very scarce.